src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -346,6 +348,8 @@ class WelcomePage extends Disposable {
 				}
 			}
 		}));
+		// {{SQL CARBON EDIT}} - Add onWelcomePageOpen event
```

Comment:
Put this on the same line as the added line

Reply:
Thanks done